src/main/java/org/hydev/mcpm/client/database/ListAllInteractor.java

```diff
@@ -0,0 +1,52 @@
+package org.hydev.mcpm.client.database;
+
+import org.apache.commons.lang3.NotImplementedException;
+import org.hydev.mcpm.*;
+import org.hydev.mcpm.client.models.PluginYml;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class ListAllInteractor {
+    LocalPluginTracker localPluginTracker = new LocalPluginTracker();
```

Comment:
```suggestion
    private final PluginTracker localPluginTracker;
```

Take it as part of the constructor!!